### PR TITLE
Add a non regression test for #2285 #2286

### PR DESCRIPTION
**Pull request by antograssiot**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

Edit: Adding test decrease coverage... of course 😈

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -183,7 +183,7 @@ jobs:
             mkdir -p build/logs/tmp build/cov
             for f in $(find features -name '*.feature' -not -path 'features/main/exposed_state.feature' | circleci tests split --split-by=timings); do
               _f=${f//\//_}
-              FEATURE="${_f}" phpdbg -qrr vendor/bin/behat --profile=coverage --suite=default --tags=~@postgres --format=progress --out=std --format=junit --out=build/logs/tmp/"${_f}" "$f"
```
Comment (dunglas) [inline .circleci/config.yml]: Why is this change needed?

Comment (dunglas) [reply .circleci/config.yml]: (It's probably this change that decreases the coverage)

File: .travis.yml [hunk 2]
```diff
@@ -52,4 +52,4 @@ script:
       vendor/bin/behat --suite=default --format=progress;
     fi
   - tests/Fixtures/app/console api:swagger:export > swagger.json && npx swagger-cli validate swagger.json && rm swagger.json
-  - tests/Fixtures/app/console api:swagger:export --yaml > swagger.yaml && npx swagger-cli validate --no-schema swagger.yaml && rm swagger.yaml
```
Comment (dunglas) [inline .travis.yml]: Why is this option needed?

File: .circleci/config.yml (hunk 1 above)
Comment (antograssiot) [reply .circleci/config.yml]: The coverage was already decreasing before this change.
I just saw that it was useless as the tags are defined in the [behat.yml.dist](https://github.com/api-platform/core/blob/master/behat.yml.dist#L61)
that's basically what I did a while ago, see https://github.com/api-platform/core/commit/71e37cbd17b17f21d36c9e52a143ecef0db9f8f5

I can revert if you prefer

**Comment by dunglas** [comment]
Thanks @antograssiot
